source/03_required_vlrs.txt

```diff
@@ -132,11 +132,11 @@ Programmatically, the data can be structured as follows:
     wKeyDirectoryVersion = 1; // Always
     wKeyRevision = 1;         // Always
     wMinorRevision = 0;       // Always
-    wNumberOfKeys        // Number of sets of 4 unsigned shorts to follow
+    wNumberOfKeys        // Number of sets of 4 uint16_ts to follow
```

Comment:
This needs an apostrophe of something.

Maybe "Number of keys (uint16_t[4]) to follow."

Reply:
Right, I guess the search-and-replace needed a bit of cleanup here... I've pushed a new wording in aee9e9e.